ops/Dockerfile

```diff
@@ -13,7 +36,7 @@ LABEL org.opencontainers.image.source=https://github.com/CMSgov/beneficiary-fhir
 # * <https://github.com/moby/moby/issues/28614>: Also some useful info.
 RUN amazon-linux-extras enable java-openjdk11 && \
     yum clean metadata && \
-    yum install -y sudo unzip selinux-policy systemd && \
+    yum install -y sudo unzip selinux-policy systemd java-11-openjdk-devel && \
```

Comment:
The project is Java based, so it's not like Java is a foreign concept here, but installing the `openjdk-devel` package here _is_ a new thing. As far as `does this PR add any new software dependencies`, what counts as a `new software dependency`? Like does this not count because it's a distro package? Or should it count because it's something that wasn't installed prior to this PR?
(This is less about your PR specifically and more me trying to understand the contours here)

Reply:
@mharvey Hey, great questions. Let me see here...

1. `java-11-openjdk-devel` is already requested and installed by the `bfd-server` role
2. this package is a superset of `java-11-openjdk`, which is requested by the `bfd-pipeline` and `bfd-db-migrator` roles
3. we're just changing _when_ this is installed for testing ansible–it's frontloaded
4. initially, usage of a separate postgresql container added a bit of time, but this change of effectively caching the installation of the jdk, it's practically a wash

I don't think there's anything untoward going on here.